examples/access-control-migration/src/repositories/project.repository.ts

```diff
@@ -20,13 +20,15 @@ export class ProjectRepository extends DefaultCrudRepository<
 > {
   public readonly user: BelongsToAccessor<User, typeof Project.prototype.id>;
 
+  public readonly owner: BelongsToAccessor<User, typeof Project.prototype.id>;
+
```

Comment:
There are several relations. If I read it correctly, `Project` only has a belongsTo relation `owner`. 
Then I think `public readonly user: BelongsToAccessor<User, typeof Project.prototype.id>;` can be deleted.